lib/proxy/index.js

```diff
@@ -25,6 +29,144 @@ HTTPS_AGENT.maxSockets = 100;
 process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
 
 /**
+ * Ordered list of mutators, validators, parsers, and other middleware for a proxy or reverse proxy.
+ * @param {Object} this_obj – A `this` Object to bind to.
+ * @param {Boolean} is_reverse_proxy - A falsey value (default) results in only forward proxy middleware being
+ * returned. A truthy value results in only reverse proxy middleware being returned.
+ * @returns {Object[]} middleware_list - Array of anonymous objects with `Object.value` containing an object evaluated
+ * using the passed in parameters. `Object.(forward|reverse)_proxy === true` iff the middleware should be used for a
+ * forward or reverse proxy, respectively. Order matters!
+ */
+function getMiddlewareList(this_obj, is_reverse_proxy) {
```

Comment:
@ryanbreen In a major update, connect() removed all middleware that it owned. It's expected that developers will use other packages to supply middleware objects to apply via `app.use()`. That's why I wrote this function to generate a list that produces a list of objects to use.